conf/distro/iot2050-debian.conf

```diff
@@ -18,6 +18,8 @@ HOSTNAME ??= "iot2050-debian"
 
 PREFERRED_VERSION_linux-iot2050 ?= "6.1.%"
 PREFERRED_VERSION_linux-iot2050-rt ?= "6.1.%"
+PREFERRED_VERSION_linux-iot2050-kbuildtarget ?= "6.1.%"
+PREFERRED_VERSION_linux-iot2050-rt-kbuildtarget ?= "6.1.%"
```

Comment:
To keep things a bit more maintainable, I would suggest this pattern:
https://gitlab.com/cip-project/cip-core/isar-cip-core/-/blob/master/conf/distro/cip-core-common.inc

Reply:
Acked and revised. BTW, sorry for the confusion, the kbuildtarget missing only confuse isar while choosing the correct kbuildtarget version, the upstream recipe is parsed regardless. This is because the "AUTOREV" in the upstream recipe. I append a new commit to address it, by masking the upstream recipe.